### PR TITLE
Unification of file names for BeagleBone.

### PR DESCRIPTION
**Pull request by DominikAdamski**
Different names for each release of Debian unables upgrading from one Debian version to another. Unification of names eliminates this problem.

Schema of unification:
/boot/vmlinuz_kernel_version -> /boot/vmlinuz
/boot/dtbs/kernel_version/am335x-boneblack.dtb -> /boot/dtbs/am335x-boneblack.dtb

Signed-off-by: Dominik Adamski <adamski.dominik@gmail.com>

### CODE REVIEW

File: bbb-convert-stage-5.sh [hunk 1]
```diff
@@ -215,6 +213,13 @@ install_files() {
   sudo ln -fs /sbin/fw_printenv $rootfs_dir/usr/bin/fw_printenv
   sudo ln -fs /sbin/fw_setenv $rootfs_dir/usr/bin/fw_setenv
 
+  #Create links for grub
+  local current_directory=$(pwd)
```
Comment (apodogrocki) [inline bbb-convert-stage-5.sh]: I would avoid changing directories here. You have available full paths for mounted boot & rootfs partitions. Just use them...

Comment (DominikAdamski) [reply bbb-convert-stage-5.sh]: I need to create relative symbolic links. Absolute symbolic links does not work under different root file system.

Comment (DominikAdamski) [reply bbb-convert-stage-5.sh]: done

File: bbb-convert-stage-5.sh [hunk 2]
```diff
@@ -215,6 +213,9 @@ install_files() {
   sudo ln -fs /sbin/fw_printenv $rootfs_dir/usr/bin/fw_printenv
   sudo ln -fs /sbin/fw_setenv $rootfs_dir/usr/bin/fw_setenv
 
+  #Create links for grub
+  sudo ln -sf /boot/dtbs/$linux_version/am335x-boneblack.dtb $rootfs_dir/boot/dtbs/am335x-boneblack.dtb
+  sudo ln -sf /boot/vmlinuz-$linux_version  $rootfs_dir/boot/vmlinuz
```
Comment (mirzak) [inline bbb-convert-stage-5.sh]: Please use generic names here. e.g:

/boot/dtb
/boot/kernel

I would expect that we will have this problem on more boards running Debian, and no point in hard-coding it to "bbb".

Comment (mirzak) [reply bbb-convert-stage-5.sh]: And the kernel image format will differ between boards.

Comment (DominikAdamski) [reply bbb-convert-stage-5.sh]: done

**Pull request merged**